### ISSUE TITLE
Sharing: Connections page not accessible via keyboard

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#423]
First reported by @kellychoffman. The Sharing page skips past two main action links: Expanding and Connecting/Disconnecting. Meaning, if a user were to only use their keyboard on this page, they are pretty much out of luck.

The items it hits on the page are in green, items that are skipped over are in red:

![sharing-1](https://cloud.githubusercontent.com/assets/11487924/11324290/c16d82da-90fb-11e5-8266-073275db5500.png)

![sharing-2](https://cloud.githubusercontent.com/assets/11487924/11324289/c16ad01c-90fb-11e5-948d-303b34392086.png)

![sharing-3](https://cloud.githubusercontent.com/assets/11487924/11324303/facf027e-90fb-11e5-8ce6-243fe32597cc.png)

**Comment by lancewillett** [comment]
This is working as expected now.

Screencast to show how "Connect" button is now in the tab index, and accessible via keyboard (testing with Mac-n-Chrome and wpcalypso).

![keyboard-sharing](https://cloud.githubusercontent.com/assets/66797/14359373/f4fceecc-fca5-11e5-9244-1bb956d106c8.gif)
